docs/infrastructure/administration.md

```diff
@@ -380,27 +77,11 @@ to emphasise the need to [connect using TLS](#connecting-to-rds-instances).
     provide the hostname of the replica and not the master - or their queries will run on
     the instance used by Treeherder, affecting its performance.
 
-When tables are added/removed, an updated set of grant statements should be generated using
-the following SQL:
-
-```sql
-SELECT CONCAT('GRANT SELECT ON ', table_schema, '.', table_name, ' to ''myuser'';') AS grant_stmt
-FROM information_schema.TABLES
-WHERE table_schema = 'treeherder'
-AND table_name NOT REGEXP 'django_|auth_';
-```
-
-<!-- prettier-ignore -->
-!!! note
-    For new tables the appropriate `GRANT SELECT` statement will need to be manually run for existing
-    read-only accounts (this is particularly important for the `redash` user).
-
 ### Changing MySQL passwords
 
-To change the password for a MySQL account on an RDS instance:
+To change the password for a MySQL account:
```

Comment:
Is this still relevant? I'd be surprised if this is something we can do still.

Reply:
No, we can't do this anymore but Jeremy wanted some of these things documented. I can clarify that only they can do this.